src/main/java/me/svdev/mscategory/model/response/CategoriesResponse.java

```diff
@@ -0,0 +1,15 @@
+package me.svdev.mscategory.model.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import me.svdev.mscategory.model.dto.CategoryDto;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+public class CategoriesResponse {
```

Comment:
[Optional] - if you return 1 DS you can return it directly without creating any wrapper classes